src/main/java/com/porotov/articleservice/controller/ArticleController.java

```diff
@@ -0,0 +1,87 @@
+package com.porotov.articleservice.controller;
+
+import com.porotov.articleservice.dto.ArticleDTO;
+import com.porotov.articleservice.model.Article;
+import com.porotov.articleservice.service.articleService.ArticleService;
+import jakarta.servlet.http.HttpServletRequest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+/**@author Alexandr Porotov
+ * @version  pre-alpha
+ * */
+@RestController
+@RequestMapping("/api")
+public class ArticleController {
+
+    private final ArticleService articleService;
+
+    private static final Logger log = LoggerFactory.getLogger(ArticleController.class.getName());
+
+    @Autowired
+    public ArticleController(ArticleService articleService){
+        this.articleService = articleService;
+    }
+
+    @GetMapping("/articles")
```

Comment:
If you use` /articles` in every method, maybe will be better move it to ` /api/articles`